### ISSUE TITLE
Tips: Try adding contextual videos to tips

### ISSUE THREAD

**Issue by kjellr** [WordPress/gutenberg#16463]
Pulling this out from https://github.com/WordPress/gutenberg/issues/16315#issuecomment-508236074:

> Tips would be extra useful if we were able to include some sort of graphic or video too. For instance, in the Block Library, maybe these could be accompanied by a brief video introduction and walkthrough for inserting a block: 
> 
> <img width="963" alt="Inserter-Tip" src="https://user-images.githubusercontent.com/1202812/60621377-83efa580-9dab-11e9-9009-6e19bdee9555.png">
> 
> Clicking the thumbnail would open that content up in a modal: 
> 
> 🖥 [**Prototype**](https://www.figma.com/proto/ctlWVC9KjYCLQlxTMAm81OT2/Tips-Iteration?node-id=93%3A30&viewport=501%2C400%2C0.4877883195877075&scaling=min-zoom)
> 
> Of course, we'd need that video content to be created + updated. But I do think it would be incredibly useful for folks. 

In general, many of us seem to think this would be a positive addition. A clear dependency here is the videos themselves — I'm not sure what the next steps would need to be to make those happen.

**Comment by mapk** [comment]
Videos!!!! would be awesome. In [today's design chat in slack](https://wordpress.slack.com/archives/C02S78ZAL/p1562689458185500), we discussed this a bit.

* Videos work better than gifs because they can be paused, or restarted.
* Videos should be generic enough that don't require continuous maintenance.

There were thoughts around showing these videos illustratively. This way if the UI changed a bit, they wouldn't need updating to reflect the finer pixels. 

Basically, we'll need someone that has a desire to build these and maintain them. I think a great place to start would be a list of videos needed. Here are a couple of ideas:

1. How to use the Block Library.
2. How to explore the options slideout.
3. How to use specific, more complex blocks.

What else?

**Comment by mapk** [comment]
I'll also add that a wizard tutorial might be another approach that can be explored. This, depending on how detailed, might work well paired with individual tips and videos.

**Comment by chrisvanpatten** [comment]
It’s also worth discussing requirements around video hosting and the impact that could have, and also the impact on external requests to third party servers (which some privacy conscious folks may not appreciate, see also Google Fonts discussions).

There are also of course a11y concerns (although I’m not at all qualified to discuss that aspect), but worth noting all the same.

**Comment by paaljoachim** [comment]
What about making it possible for the web designer/developer to add in their own videos instead of having the default ones? As I would be able to make videos for my clients. That the tips would be able to be customized by the web person if needed.

**Comment by mapk** [comment]
@paaljoachim if videos become a part of the Tips component, then I can totally see this happening by way of a plugin. https://github.com/WordPress/gutenberg/issues/16594 mentions the possibility for plugin authors to add to these Tips with their own.

**Comment by mapk** [comment]
I'm closing this out for now since...

1. We've hit a blocker on the video hosting.
2. We've done a lot of NUX work lately and included more Help links throughout.
